python/core/auto_generated/pointcloud/qgspointcloudrenderer.sip.in

```diff
@@ -48,6 +48,78 @@ Returns a reference to the context's render context.
 %End
 
 
+
+    void setElevationPainter( QPainter *painter );
```

Comment:
I'd SIP_SKIP all these -- I don't think they should be part of stable API.

Eg. the elevation painter would need to move to QgsRenderContext for use outside of point clouds.